webpack/components/extensions/HostDetails/Tabs/ErratumExpansionContents.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  TreeView,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const ErratumExpansionContents = ({ erratum }) => {
+  const {
+    bugs, cves, packages,
+    module_streams: moduleStreams,
+  } = erratum;
+  const [activeItems, setActiveItems] = useState(null);
+  const options = [
+    {
+      name: __('Bugs'),
+      id: 'bugs',
+      children: bugs.map(bug => ({ name: bug.bug_id, id: bug.bug_id, ...bug })),
+    },
+    {
+      name: __('CVEs'),
+      id: 'cves',
+      children: cves.map(cve => ({ name: cve.cve_id, id: cve.cve_id, ...cve })),
+    },
+    {
+      name: __('Packages'),
+      id: 'packages',
+      // packages is just a list of strings
+      children: packages.map((packageName, idx) => ({ name: packageName, id: idx })),
+    },
+    {
+      name: __('Module streams'),
+      id: 'module_streams',
+      // module streams should already have name and id fields, so no need to map
+      children: moduleStreams,
+    },
```

Comment:
The Patternfly `<TreeView>` component requires an array of objects with each having a `name` and `id` field. 
As far as I could tell from reading the `rabl` files, the module stream objects within a host erratum should have `name` and `id` fields, already so there's no need to transform the data here.  But I wasn't able to test this, since none of my RHEL 7 errata had any module streams. @jlsherrill or @ianballou does this look correct to you?